MVP Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -52,6 +52,10 @@ class FeedUISnapshotTests: XCTestCase {
 
 private extension FeedViewController {
 	func display(errorMessage: String) {
-		fatalError("Must be implemented - follow the MVC solution as a guide")
+		errorView?.show(message: errorMessage)
+	}
+	
+	private var errorView: ErrorView? {
+		tableView.tableHeaderView as? ErrorView
```

Comment:
I recommend using the `errorView` test helper here. Otherwise, changing the storyboard view hierarchy may break the code here.

```suggestion
		errorView?.show(message: errorMessage)
	}
```